lib/Models/Workbench.ts

```diff
@@ -167,14 +166,15 @@ export default class Workbench {
    *
    * @param item The item to add to or remove from the workbench.
    */
-  public async add(item: BaseModel | BaseModel[]): Promise<void> {
+  public async add(
+    item: BaseModel | BaseModel[],
+    index: number = 0
+  ): Promise<void> {
```

Comment:
Added index parameter to specify the position in the workbench where we want to insert the item.